src/crypto/lib/encryption.ts

```diff
@@ -0,0 +1,27 @@
+const crypto = require('crypto');
+
+//Encryption
+export const _encryptData = function (data: Object, cipherKey: Object): Promise<Buffer> {
+    
+  return new Promise((resolve) => {
+    let iv = crypto.randomBytes(16);
+    const cipher = crypto.createCipheriv('aes256', cipherKey, iv);
+    const encryptedData: Buffer = Buffer.concat([iv, cipher.update(data), cipher.final()]);
```

Comment:
The crypto package should be imported in `utils` and the aes encryption can happen in `utils`. You can move the above 3 statements to utils and create a wrapper function called `aesEncryption`.